src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card__wrapper {
+  margin: 0 16px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
```

Comment:
Don't fix container size (if there is no such a requirement)
Let the content size dictate it. To avoid overflow or accidental scroll bar

Reply:
Okey, I'll correct this moment